src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -7,19 +7,22 @@
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
     <UseOpenKey Condition="'$(UseOpenKey)'==''">true</UseOpenKey>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'netcoreapp1.1'">.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
```

Comment:
Since `ValueTuple` will only ship in CoreCLR's corlib in 1.2, I probably need a `TargetGroup` and `NuGetTargetMoniker` for 1.2. 

@tarekgh @jkotas Should I just go ahead and use 1.2 here, or does some plumbing first need to be created to declare those?